etc/flyway/database/V2__create_and_populate_countries.sql

```diff
@@ -0,0 +1,250 @@
+CREATE TABLE country (
+  id SERIAL PRIMARY KEY,
+  iso VARCHAR(2) NOT NULL,
+  name VARCHAR(80) NOT NULL,
+  nicename VARCHAR(80) NOT NULL,
+  iso3 VARCHAR(3) DEFAULT NULL,
+  numcode SMALLINT DEFAULT NULL,
+  phonecode INT NOT NULL
+);
+
+INSERT INTO country VALUES
+(1, 'AF', 'AFGHANISTAN', 'Afghanistan', 'AFG', 4, 93),
```

Comment:
Insert only a few countries (5-10) for now. Store all inserts to some file in etc.